google_symptoms/params.json.template

```diff
@@ -26,9 +26,13 @@
     "dynamic": {
       "ref_window_size": 7,
       "smoothed_signals": [
-        "ageusia_smoothed_search",
-        "sum_anosmia_ageusia_smoothed_search",
-        "anosmia_smoothed_search"
+        "s01_smoothed_search",
+        "s02_smoothed_search",
+        "s03_smoothed_search",
+        "s05_smoothed_search",
+        "s06_smoothed_search",
+        "s08_smoothed_search",
+        "scontrol_smoothed_search"
```

Comment:
Please apply the same changes to the production params file, [here](https://github.com/cmu-delphi/covidcast-indicators/blob/20da1c0c6da1f2886e87b88b671f8247e8a36fad/ansible/templates/google_symptoms-params-prod.json.j2).

The API key is already accessible to the production params, so we don't need to do anything there.

Reply:
Thanks for the pointer! I didn't know I had access to the production params file.
I just pushed the updated file.